pkg/resources/collector/templates.go

```diff
@@ -101,6 +102,49 @@ journald/k3s:
   units: [ "k3s" ]
   directory: {{ . }}
 `))
+
+	templateKubeAuditLogs = `
+filelog/kubeauditlogs:
+  include: [ /var/log/kube-audit/*.log ]
```

Comment:
We should probably be aware that this hard-coded value may not always be correct https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#log-backend. 

I doubt there is a way to programmatically discover which path is in use by the apiserver, but figure I'd ask anyways

We should check (if you haven't already) that this prefix is the default for all the Kubernetes distributions we currently support with logs, EKS, RKE1/2, K3s

Reply:
Agreed. From what I've seen in the docs, there's no "default" audit log location and it does not pair that well across the distributions.

I've seen [this small "how to" from SUSE](https://www.suse.com/support/kb/doc/?id=000021022) showing how one can activate audit logs with rancher-logging. In this case, if no LogPath is provided in the CollectorConfigCRD, it uses the article's path. 

I confess I'm still confused about how the reconciling process works for the logging plugin, so I'm not sure if the CRD is really reconciled and, thus, if the LogPath does get picked up somewhere.